### ISSUE TITLE
DietPi-Software | rclone: Install fails on ARMv6 RPi models

### ISSUE THREAD

**Issue by eddiermar** [MichaIng/DietPi#5601]
#### Required 
- DietPi version | 8.6.1 
- SBC model | Raspberry pi b+

#### Additional Information (if applicable)
- Software title | Rclone

- Bug report ID | af2861c7-6caf-442a-b555-fbf26cc12607

#### Steps to reproduce

1. go to dietpi-software
2. try to install rclone
3. installation fails

#### Expected behaviour

- Rclone is installed

#### Actual behaviour

- I think the rclone installer is wrongly set because it tries to install the amd64 version.

#### Extra details

 
│ APT install for: ./rclone-v1.58.1-linux-amd64.deb
│  - Command: apt-get -y --allow-change-held-packages install
│ ./rclone-v1.58.1-linux-amd64.deb

**Comment by MichaIng** [comment]
Many thanks for your report.

Indeed a bug our end with a false/missing condition statement. Sadly difficult to push a live patch for this.

For now:
```bash
cd /tmp
curl -fL 'https://github.com/rclone/rclone/releases/download/v1.58.1/rclone-v1.58.1-linux-arm.deb' -o rclone.deb
apt install ./rclone.deb
rm rclone.deb
```

**Comment by MichaIng** [comment]
Fixed with: https://github.com/MichaIng/DietPi/commit/bacf34c

**Comment by MichaIng** [comment]
Fix can be tested in beta: #5643
